docker-compose.yml

```diff
@@ -27,8 +27,8 @@ services:
       test: ["CMD", "/etc/init.d/mysql", "status"]
       interval: 30s
       timeout: 3s
-      retries: 3
-      start_period: 10s
+      retries: 5
+      start_period: 90s
```

Comment:
Are you developing on macOS or Linux? While this is just for developers so I'm not really concerned with bumping this, I want to make sure there's not a bug with the container.

Personally, I noticed that when I switched from developing on Linux to macOS, I saw more startup errors. Saw some mentions online of needing a different MySQL container but didn't dig into this more.

Reply:
Yes, the e2e-test.sh works on linux, but I see some platform-related errors when running it on mac, I suspect I need to remove the `platform:` specification and somehow find another multiplatform image for the gcp-pubsub-emulator

Reply:
@bobcallaway thanks,  #2406 helps out, though I'm still having some perhaps unrelated port-forwarding issues on mac with rancher-desktop.

Reply:
can you share your specific commands that you're running which aren't working?